### DISCUSSION TITLE
Android Auto Help - no display

### DISCUSSION THREAD

**Discussion by coacharnold1** [CappielloAntonio/tempo#192] [Q&A]
Not sure if this elevates to a bug issue so i'll post here and you folks tell me.  

I'm on Android 13 build T1T0S33.3-45-23-12   on a moto G power 
using The current release of Tempo 3.7  from F-Droid

Temo shows up on my Auto Car screen,  but when I start it all i get i a blank screen.  If I go to my connected phone and press play in the app,  i can play through android-auto no problem,   Is there a trick I'm missing getting the car display to work with Tempo?  Perhaps something in the server settings?  Client Settings?

Thanks for any help .....   regardless i love this player?

Tim 
NJ USA

**Comment by rubytube76** [comment]
Did you allowed unknown sources in Android Auto settings?

Search android auto in your phone settings.
Click a lot of times in version field to activate developer settings.
Go to the menu in Android auto settings and set developer settings (or similar)
Activate Unknown sources check.

Hope it helps

Ruby

**Comment by coacharnold1** [comment]
What your answer is for is help installing ....  Install is fine,  well aware of how to install unkonw sources.  THis is a great player,  In android auto it works (in my car)  but nothing is displayed on my car screen.   All I get is a black screen.   Music plays through,  but it has to be controlled from the phone.....  is this correct??

**Reply by rubytube76** [reply]
No, it is not for installing, is to allow the app to interact with Android Auto.

Please re-read my message, it's all about "Android Auto" settings, not Android settings.

Hope it helps

**Reply by rubytube76** [reply]
Maybe this way is easier:

Go to Settings > Apps > **Android Auto** > 'Additional settings in the app' > Scroll down to "Version and permission info" and tap that box 10 times or until it tells you you're in developer mode.
Then click the ... menu in the top right > Developer settings > Check "Unknown sources".

**Reply by krononox** [reply]
Hi

it's the same for me (and "unknown sources" is enabled).
As coacharnold1, Temo shows up on car screen, but when i open it, there is a black screen.

Android 14
android auto v 11.5.641024
Tempo v 3.8.1

**Reply by coacharnold1** [reply]
Thais exactly what I have -  "unknown sources" has been enabled  in android auto since I installed it, and I still get a black screen.  SOund comes through fine, but no screen GUI.  I still use it so I can play music while using the map ... but I would love to get the screen controls to work.

**Comment by dror3go** [comment]
Same issue for me using GrapheneOS.
Enabling unknown sources was required to even see the Tempo icon in the car and listening to music, but the screen is black with no content.
